Remove hugo-pacman

onedrawingperday opened the pull request:
This theme contains third party tracking.

The author never responded to the corresponding issue: https://github.com/coderzh/hugo-pacman-theme/issues/26

coderzh commented:
Hi,  third party tracking has been removed, could you add Pacman back?

https://github.com/coderzh/hugo-pacman-theme/commit/ccb3b854e484852fb9f6527eb75c387931d1eeb7

onedrawingperday commented:
Hi @coderzh 

Thank you for addressing the issues regarding 3rd party tracking and mixed content.

Your theme will be added back once #587 is merged.
